src/test/java/com/geronimo/MessageServiceTest.java

```diff
@@ -33,33 +33,30 @@
     private EntityManager entityManager;
 
     private User author;
+    private User badUser;
 
     @Transactional
     @Before
     public void before() {
         userService.deleteUserByUsername("nice_user");
+        userService.deleteUserByUsername("not_nice_user");
         entityManager.flush();
 
         author = userService.saveUser(new User("nice_user", "nice_password"));
+        badUser = userService.saveUser(new User("not_nice_user", "neither_nice_password"));
     }
 
     @Transactional
     @After
     public void after() {
+        userService.deleteUserById(badUser.getId());
```

Comment:
Right after the line deleteByUserId, deleteUser method is used. Looks a little bit weird :)

Why not to do this the same way?

Reply:
Y O L O